CMakeLists.txt

```
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 2.8.12...3.20)
 
-project(enet)
+project(enet LANGUAGES C)
```

```suggestion
project(enet C)
```
Since the minimum CMake version is still set to 2.8, it might not support the new style (it's not documented [here](https://cmake.org/cmake/help/v2.8.12/cmake.html#command:project) at least), hence I'd suggest to use the old style, which is still supported by latest CMake as well.

---

Sounds reasonable, thanks!